### PR TITLE
DTGB-562: Added file type to the CTA block.

### PR DESCRIPTION
**Pull request by meirege**
## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] I have read the **CONTRIBUTING** document.
- [ ] I have updated the documentation accordingly.
- [ ] I have updated the style guide CHANGELOG accordingly.
- [ ] I have updated the gent_base CHANGELOG accordingly.
- [ ] I have ran gulp axe on the compiled files and found no errors.

### CODE REVIEW

File: templates/contrib/paragraphs/cta/paragraph--call-to-action.html.twig [hunk 1]
```diff
@@ -32,8 +32,8 @@
           {% if primary_type == 'document' %}
             {% if content.field_primary_action_document|render|striptags|trim is not empty %}
               <a href="{{ primary_document_file_uri }}" class="standalone-link button" download>{{ content.field_primary_action_title|render|striptags|trim }}</a>
```
Comment (delrueba) [inline templates/contrib/paragraphs/cta/paragraph--call-to-action.html.twig]: Isn't it either a button or a standalone-link? Not both.

File: templates/contrib/paragraphs/cta/paragraph--call-to-action.html.twig [hunk 2]
```diff
@@ -32,8 +32,8 @@
...
-              {% if primary_document_size %}
```
Comment (delrueba) [inline templates/contrib/paragraphs/cta/paragraph--call-to-action.html.twig]: I wouldn't put these in the same conditional. 
If file size is missing, i'm still interested in the type.

**Review by delrueba — CHANGES_REQUESTED** [review]
Missing variable documentation in the twig file.

File: templates/contrib/paragraphs/cta/paragraph--call-to-action.html.twig (hunk 2 above)
Comment (meirege) [reply templates/contrib/paragraphs/cta/paragraph--call-to-action.html.twig]: If file size is missing it means no file was uploaded... so you don't get a type either... :)

Comment (delrueba) [reply templates/contrib/paragraphs/cta/paragraph--call-to-action.html.twig]: Allright, maybe add to the documentation that these are linked.

Comment (meirege) [reply templates/contrib/paragraphs/cta/paragraph--call-to-action.html.twig]: Done.

**Review by delrueba — APPROVED** [review]

**Pull request merged**